src/lib/stitching/gravity/schema.ts

```diff
@@ -49,6 +50,14 @@ export const executableGravitySchema = () => {
     "ArtistSeriesConnection",
   ]
 
+  if (config.USE_UNSTITCHED_SECOND_FACTORS_SCHEMA) {
+    duplicatedTypes.push("SecondFactor")
```

Comment:
This list will eventually grow to include all the mutation-related types as well, inputs, and so on.